frontend/coprs_frontend/coprs/views/webhooks_ns/webhooks_general.py

```diff
@@ -347,6 +351,9 @@ def custom_build_submit(copr, package, copr_dir=None):
         storage.delete()
         return "BUILD_REQUEST_ERROR\n", 500
 
+    user_agent = flask.request.headers.get('User-Agent')
```

Comment:
this can be `None`. We need to either add here check `if user_agent is None then user_agent = "Not Set"` or maybe better do this check inside add_webhook_history_record and s/user_agent="Not Set"/user_agent=None/

Reply:
done. added a check inside `add_webhook_history_record`